packages/cli/__tests__/zoslogs/__integration__/list/cli.zos-logs.list.logs.integration.test.ts

```diff
@@ -0,0 +1,38 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ITestEnvironment } from "../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestEnvironment } from "../../../../../../__tests__/__src__/environment/TestEnvironment";
+import { runCliScript } from "../../../../../../__tests__/__src__/TestUtils";
+
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+
+describe("zos-logs list", () => {
+    // Create the unique test environment
+    beforeAll(async () => {
+        TEST_ENVIRONMENT = await TestEnvironment.setUp({
+            testName: "zos_log_get",
+            skipProperties: true
+        });
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(TEST_ENVIRONMENT);
+    });
+
+    it("should display the help", async () => {
+        const response = runCliScript(__dirname + "/__scripts__/list/help.sh", TEST_ENVIRONMENT);
```

Comment:
If the folder is renamed as suggested in above comment, this path would change from "list/help.sh" -> "logs/help.sh"